src/style.css

```
@@ -0,0 +1,113 @@
+@import "./stars.css";
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
+
+.card__img-iMac {
+  margin-bottom: 40px;
+  height: 134px;
+  width: 160px;
+  align-self: center;
+}
+
+.card__name {
+  display: flex;
+  justify-content: center;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__name,
+.card__rating-reviews,
+.card__prise-amount {
+  color: #060b35;
```

don't forget to create variables for all colors
```suggestion
  color: var(--color-some);
```